docs/recipes/managing_multiple_clusters.adoc

```diff
@@ -0,0 +1,164 @@
+╰────┴─────────────────────────┴──────────────────────────────────────────────────────────────────────────────────────┴─────────┴──────────────────────────┴───────────────────────┴───────────────────────────┴──────────────┴─────────────┴─────╯
+```
+
+This gives us plenty of information, but sometimes it can be a bit difficult to read.
+We can make things much easier with some simple reformatting.
+To focus on the free memory that each cluster has, we can https://www.nushell.sh/commands/docs/select.html[select] just the relevant columns:
+
+```
+👤 Charlie 🏠 local in 🗄 travel-sample._default._default
+> nodes --clusters * | select cluster memory_free
+╭────┬─────────────────────────┬─────────────╮
+│  # │         cluster         │ memory_free │
+├────┼─────────────────────────┼─────────────┤
+│  0 │ local                   │  2841657344 │
+│  1 │ local                   │  2842959872 │
+│  2 │ local                   │  2843160576 │
+│  3 │ bronzegeraldjaysussman  │ 15518982144 │
+│  4 │ bronzegeraldjaysussman  │ 15518420992 │
+│  5 │ bronzegeraldjaysussman  │ 15501099008 │
+│  6 │ obligingfaronmoller     │ 14538944512 │
+│  7 │ obligingfaronmoller     │ 14559510528 │
+│  8 │ obligingfaronmoller     │ 14565412864 │
+│  9 │ meekjoycekreynolds      │ 15491842048 │
+╰────┴─────────────────────────┴─────────────╯
+```
+
+We could then convert the `memory_free` column from bytes to gigabytes as follows:
+
+[options="nowrap"]
+```
+👤 Charlie 🏠 local in 🗄 travel-sample._default._default
+> nodes --clusters * | each {|n| $n | update memory_free ($n.memory_free * 1B)} | select cluster memory_free
+╭───┬────────────────────────┬─────────────╮
+│ # │        cluster         │ memory_free │
+├───┼────────────────────────┼─────────────┤
+│ 0 │ local                  │     2.6 GiB │
+│ 1 │ local                  │     2.6 GiB │
+│ 2 │ local                  │     2.6 GiB │
+│ 3 │ bronzegeraldjaysussman │    14.5 GiB │
+│ 4 │ bronzegeraldjaysussman │    14.5 GiB │
+│ 5 │ bronzegeraldjaysussman │    14.4 GiB │
+│ 6 │ obligingfaronmoller    │    13.5 GiB │
+│ 7 │ obligingfaronmoller    │    13.6 GiB │
+│ 8 │ obligingfaronmoller    │    13.6 GiB │
+│ 9 │ meekjoycekreynolds     │    14.4 GiB │
+╰───┴────────────────────────┴─────────────╯
+```
+
+We do this by iterating over each node and https://www.nushell.sh/commands/docs/update.html[updating] the value in the `memory_free` column by multiplying the current value by nushell's inbuilt https://www.nushell.sh/book/types_of_data.html#file-sizes[File Size] datatype.
+
+While it is somewhat useful to know the free memory that each cluster has, it'd be more useful for our healthcheck to know the memory utilization for each cluster.
+Unfortunately the info returned by `nodes` does not include the memory utilization, however there are two columns that can be used to calculate this: `memory_free` and `memory_total`.
```

Comment:
Again, quite cool, but it might be neat to fit it into a narrative.  LIke it's useful to find a test cluster with space to create a bucket, yadda, yadda.

Reply:
The narrative started off as performing health checks on the cluster, then once the health is understood we can use the resource info to choose where to allocate a bucket. I've changed this to link it more heavily to the health check then I have the bit about the bucket later on: 

```
Now that we understand the resources being used by our cluster we can use this information to aide us when deploying a new bucket.
Imagine that we want to create a 1GB bucket on any one of our clusters.
```